*** iset.mm	2021-10-20 17:47:14.148746900 +0200
--- iset.mm	2021-10-20 17:50:22.193298400 +0200
***************
*** 345,350 ****
--- 345,352 ----

  $)

+   $c TOP $.
+
    $( Declare the primitive constant symbols for propositional calculus. $)
    $c ( $.  $( Left parenthesis $)
    $c ) $.  $( Right parenthesis $)
***************
*** 355,360 ****
--- 357,369 ----
    $c |- $. $( Turnstile (read:  "the following symbol sequence is provable" or
                'a proof exists for") $)

+   ${
+     $v ph $.
+     wph.TOP $f wff ph $.
+     TOP.wff $a TOP wff ph $.
+     TOP.turnstile $a TOP |- ph $.
+   $}
+
    $( Declare typographical constant symbols that are not directly used
       in the formalism, but *are* symbols we find useful when
       explaining the formalism. It is much easier to consistently use
***************
*** 530,535 ****
--- 539,545 ----
       Postulate the modus ponens rule of inference.
    $)

+
    ${
      $( Minor premise for modus ponens. $)
      min $e |- ph $.
***************
*** 9291,9296 ****
--- 9301,9312 ----

    $( Add 'setvar' as a typecode for bound variables. $)
    $( $j syntax 'setvar'; bound 'setvar'; $)
+   ${
+     $v x $.
+     vx.TOP.setvar $f setvar x $.
+     TOP.setvar $a TOP setvar x $.
+   $}
+

    ${
      $v x $.
***************
*** 9328,9333 ****
--- 9344,9354 ----

    $( Add 'class' as a typecode. $)
    $( $j syntax 'class'; $)
+   ${
+     $v A $.
+     cA.TOP.class $f class A $.
+     TOP.class $a TOP class A $.
+   $}

    ${
      $v x $.
