*** set.mm	2021-10-06 09:24:36.353718200 +0200
--- set.mm	2021-10-20 17:28:30.443844400 +0200
***************
*** 12472,12477 ****
--- 12472,12479 ----
  
  $)
  
+   $c TOP $.
+ 
    $( Declare the primitive constant symbols for propositional calculus. $)
    $c ( $.  $( Left parenthesis $)
    $c ) $.  $( Right parenthesis $)
***************
*** 12491,12496 ****
--- 12493,12505 ----
      syntax '|-' as 'wff';
      unambiguous 'klr 5';
    $)
+   ${
+     $v ph $.
+     wph.TOP $f wff ph $.
+     TOP.wff $a TOP wff ph $.
+     TOP.turnstile $a TOP |- ph $.
+   $}
+ 
  
    $( Declare typographical constant symbols that are not directly used
       in the formalism, but *are* symbols we find useful when
***************
*** 23427,23432 ****
--- 23436,23447 ----
  
    $( Add 'setvar' as a typecode for bound variables. $)
    $( $j syntax 'setvar'; bound 'setvar'; $)
+   ${
+     $v x $.
+     vx.TOP.setvar $f setvar x $.
+     TOP.setvar $a TOP setvar x $.
+   $}
+ 
  
    ${
      $v x $.
***************
*** 23466,23471 ****
--- 23481,23491 ----
  
    $( Add 'class' as a typecode. $)
    $( $j syntax 'class'; $)
+   ${
+     $v A $.
+     cA.TOP.class $f class A $.
+     TOP.class $a TOP class A $.
+   $}
  
    ${
      $v x $.
